http_service/Dockerfile

```diff
@@ -18,4 +18,4 @@ ENV CHECK_MODELS="${CHECK_MODELS}"
 
 RUN python /code/bugbug_http_service/check_models.py
 
-CMD ["gunicorn", "-b", "0.0.0.0:8000", "bugbug_http_service.app", "--preload", "--timeout", "30", "-w", "3"]
+CMD gunicorn -b 0.0.0.0:$PORT bugbug_http_service.app --preload --timeout 30 -w 3
```

Comment:
Is the other format not OK on HEroku?

Reply:
No the other format couldn't be updated as the list-based format is directly executed by Docker. This format means that it's wrapped by bash for argument parsing and the nice side-effect is that bash is doing the variable interpolation for us.